common/lib/protocol-definitions/src/summary.ts

```diff
@@ -21,41 +21,89 @@ export interface ISummaryCommitter {
     date: string;
 }
 
+/**
```

Comment:
Orthogonal to this PR - Interesting to note that `ISummaryAuthor` and `ISummaryCommiter` above are never used. We may want to look into removing them.

Reply:
Tracking by  [https://github.com/microsoft/FluidFramework/issues/10456](https://github.com/microsoft/FluidFramework/issues/10456)